src/NzbDrone.Core.Test/Languages/LanguageFixture.cs

```diff
@@ -43,7 +43,9 @@ public class LanguageFixture : CoreTest
                 new object[] { 28, Language.Thai },
                 new object[] { 29, Language.Bulgarian },
                 new object[] { 30, Language.PortugueseBR },
-                new object[] { 31, Language.Arabic }
+                new object[] { 31, Language.Arabic },
+                new object[] { 32, Language.Persian }
```

Comment:
please check the number vs other pre-existing language PRs

Reply:
I incremented the number to take #6385 into account.